ai_flow/model/task_execution.py

```diff
@@ -63,9 +74,52 @@ def __init__(self,
         self.workflow_execution_id = workflow_execution_id
         self.task_name = task_name
         self.sequence_number = sequence_number
-        self.try_number = try_number
         self.execution_type = execution_type
         self.begin_date = begin_date
         self.end_date = end_date
         self.status = status
 
+        self.try_number = 0
+
+    def get_task(self):
+        snapshot_path = workflow_execution.get_workflow_snapshot(self.workflow_execution_id)
+        workflows: List[Workflow] = workflow_utils.extract_workflow(snapshot_path)
+        name = workflow_execution.get_workflow_name_by_execution_id(self.workflow_execution_id)
```

Comment:
Is it possible to fetch workflow from metadata？

Reply:
Good point, it would be an optimazition to decrease the workload of the scheduler. I'll do this in the following pr and for now, I think it's ok to extract workflow from files on workers.